packages/sbcl.rb

```diff
@@ -0,0 +1,32 @@
+require 'package'
+
+class Sbcl < Package
+  description 'Steel Bank Common Lisp (SBCL) is a high performance Common Lisp compiler.'
+  homepage 'http://www.sbcl.org/index.html'
+  version '1.5.3'
+  source_url 'http://prdownloads.sourceforge.net/sbcl/sbcl-1.5.3-source.tar.bz2?download'
+  source_sha256 'd2a7901144d227a7194661dc3ccee28873c8d5cbbd167013660c78b9ad63640c'
+
+  binary_url ({
```

Comment:
Please remove the empty `binary_url` and `binary_sha256` blocks.

Reply:
@lloydhuang: It's fine to leave the empty blocks.

Reply:
@uberhacker It will cause the download to try and download an empty file and verify it with an empty sha256sum, instead of downloading the sources and installing as usual.